src/config/settings.py

```
@@ -96,6 +96,13 @@ def get_auto_reply_channels(self) -> list[str]:
     rag_debug_log_enabled: bool = False  # 本番ではPII漏洩リスクのためデフォルト無効
     rag_show_sources: bool = False
 
+    # Safe Browsing (URL安全性チェック)
+    rag_url_safety_check: bool = False
+    google_safe_browsing_api_key: str = ""
+    rag_url_safety_cache_ttl: int = Field(default=3600, ge=0)  # キャッシュTTL秒（0=APIレスポンスに従う）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)